docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -269,6 +293,65 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Confluent Table Description Supplier
+------------------------------------
+The confluent table description supplier uses the `Confluent Schema Registry
+<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_. to
+discover table definitions. The benefits of using the schema registry is that new tables can be defined without
+a cluster restart, schema updates will be detected and there is no need to define tables manually.
+
+========================================================== ======================================================================================================
+Property Name                                              Description
+========================================================== ======================================================================================================
+``kafka.confluent-schema-registry-url``                    List of addresses for the confluent schema registry
+``kafka.confluent-schema-registry-client-cache-size``      The maximum number of subjects that can be stored in the local cache
+``kafka.empty-field-strategy``                             How to handle struct types with no fields: ignore, add a boolean field named 'dummy' or fail the query
+``kafka.confluent-subjects-cache-refresh-interval``        The interval that the topic to subjects cache will be refreshed
+========================================================== ======================================================================================================
+
+``kafka.confluent-schema-registry-url``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The comma separated list of schema registry urls used to connect to the schema registry, for example ``http://schema-registry-1.example.org:8081,http://schema-registry-2.example.org:8081``.
+This property is required if the ``CONFLUENT`` table description supplier is used; there is no default.
+
+``kafka.confluent-schema-registry-client-cache-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The size of the local subjects cache.
```

Comment:
Can you please elaborate a bit more about what information about subject cache holds?